gp-includes/gp.php

```diff
@@ -165,4 +165,13 @@ class GP {
 	 * @var array
 	 */
 	public static $scripts = array();
+
+	/**
+	 * Model for GP_WP_Settings.
+	 *
+	 * @since 2.2.0
+	 *
+	 * @var GP_WP_Settings
+	 */
+	public static $wp_settings;
```

Comment:
This looks weird. Can we make that `$admin` and use it for all (future) stuff which is handled in the WordPress admin?

This should also only be initialized when we're in the admin, not on the front end.